project/Build.scala

```diff
@@ -35,22 +35,21 @@ object Build extends AutoPlugin {
   override def projectSettings = Seq(
     organization := org,
     // a 'compileonly' configuation
-    ivyConfigurations += config("compileonly").hide,
+//    ivyConfigurations += config("compileonly").hide,
```

Comment:
`config must be directly assigned to a val, such as `val Tooling = config("tooling")`.`
What is this setting doing?

Reply:
No idea !

Reply:
Is it ok to remove this then?